Update Readme.md

dafakias opened the pull request:
3ο ΠΑΡΑΔΟΤΕΟ

dafakias committed:
Update Readme.md

3ο ΠΑΡΑΔΟΤΕΟ

c15pati reviewed (commented):
Όσον αφορά τα παραδοτέα - το 1 είναι εντάξει.
το 2 το αλλάζεις στο 3. Εντάξει το καταλαβαίνω, αλλά μείνε εκεί. Νομίζω ότι το 3 "προσθήκη νέων λειτουργιών" άλλαξες το μέγεθος της οθόνης, όπως και την κατηγορία. Συνέχισε να το δουλεύεις θα έλεγα εγώ. 

Επίσης: 
Κάποια σχόλια σχετικά με το παιχνίδι μπορεί κάποια να είναι πρώιμα αλλά καλό είναι να υπάρχουν – Φυσικά το πρώτο πράγμα, να βάλεις μία υπόδειξη για τα controls 
και επίσης :

1.	Δεν ξέρω γιατί σταμάτησε να είναι τύπου shooting. Αλλά ότι προτιμάει ο κατασκευαστής φυσικά. Από άποψη λειτουργίας, δεν είναι ότι κάποιο έχει ανώτερα εκπαιδευτικά αποτελέσματα από τις δύο κατηγορίες.
2.   Ζωές δεν χάνονται, απλά ξεκινάει από την αρχή όταν ακουμπήσεις τους γλάρους (υπέθεσα εχθροί?)
3.	Αν αλλάξω παράθυρο καθώς έχω ανοικτό το παιχνίδι ο χαρακτήρας κολλάει και προχωράει προς μία κατεύθυνση
4.	Είναι απαραίτητο, να ξαναεμφανίζονται οι λάθος ή σωστοί αριθμοί θεωρώ. (Από άποψη Gameplay, και εκπαιδευτική – Αν τα επιλέξεις όλα και χαθούν, τότε δεν κάνουμε τίποτα)
5.	Η τάση στα Mario παιχνίδια – το αντανακλαστικό αν θέλεις είναι ότι βρίσκεις στην πίστα (ειδικά αν μοιάζει με χρυσό νόμισμα) να το μαζεύεις εκτός αν είναι εχθρός – Νομίζεις ότι έτσι όπως ξεκινάει το παιχνίδι ο παίκτης θα το έχει κατανοήσει αυτό ή θα πρέπει να διαβάσει υποδείξεις?

Γενικότερα σχόλια που μπορεί να έχεις σκεφτεί:
1.	Ο χαρακτήρας είναι καλοσχεδιασμένος αλλά πρέπει να είναι μεγαλύτερου μεγέθους
2.	Κάποιο γραφικό στο background


Αυτά προς το παρόν, νομίζω ότι είναι καλό να αφιερωθείς σε μία κατεύθυνση και να μην το εγκαταλείψεις, αλλά να προσπαθήσεις να αντιμετώσεις τα προβλήματα του. Θέλεις Platform εντάξει, θέλεις shooting πάλι εντάξει, αλλά πρέπει να είναι άρτιο στην κατεύθυνση που επέλεξες. 

Αυτά προς το παρόν, και ελπίζω σε συνέχεια.

epidrome commented:
@bgarnb @c15pati θα πρέπει να βάλετε και μια ένδειξη δεκτό-αλλαγές-άκυρο ώστε ο συντονιστής της πτυχιακής @merkourisa να αποφασίσει αν θα κάνει δεκτό το αίτημα ή θα πρέπει να γίνουν αλλαγές

bgarnb reviewed (changes requested):
Δοκίμασα το λινκ στο παιχνίδι και δεν παίζει εκτός και τώρα υπάρχει κάποι άλλο που δεν το έχεις προσθέσει. Οπότε δεν μπορώ να δω αυτά που περιγράφεις. Σε κάθε περίπτωση, καλό θα ήταν να έβαζες έστω κάποια ενδεικτική οθόνη σε κάθε νέα λειτουργία ή τροποίηση που έκανες γιατί έτσι είναι πολύ αόριστο.

dafakias commented:
Καλησπέρα, το λίνκ είναι αυτό https://dafakias.github.io/Super-Mario/ .
Δεν γνωρίζω γιατί δεν δουλεύει κάποιες φορές από το αρχείο του readme.

dafakias commented:
![image 2](https://user-images.githubusercontent.com/11212020/35782730-4b30194a-0a05-11e8-9a22-3368167cd19a.jpg)
![image 1](https://user-images.githubusercontent.com/11212020/35782731-4b5c5122-0a05-11e8-9f9a-8b938647957c.jpg)
![image 3](https://user-images.githubusercontent.com/11212020/35782732-4b85a11c-0a05-11e8-8fba-4ed79a4fd06f.jpg)
![image 4](https://user-images.githubusercontent.com/11212020/35782733-4bafb34e-0a05-11e8-9ff4-c931ea166eab.jpg)

merkourisa commented:
@dafakias  Καλησπέρα. Καλύτερα τις εικόνες που σου ζητάει η @bgarnb να τις περάσεις μέσα στην αναφορά σου και όχι σαν σχόλιο. Επίσης στο https://github.com/dafakias/Super-Mario άλλαξε το link από το https://ioniodi.github.io/Super-Mario/ σε https://dafakias.github.io/Super-Mario/ για να μην υπάρχει μπέρδεμα όταν πάμε να τρέξουμε το παιχνίδι. Τέλος θα σε συμβούλευα να ρίξεις μια ματία εδώ σε ένα παραδοτέο που έγινε πρόσφατα δεκτό https://github.com/courses-ionio/thesis/tree/master/current/2014221

bgarnb commented:
@dafakias οκ διόρθωσε το παραδοτέο σου προσθέτοντας τις εικόνες που έχεις παραπάνω και αντιστοιχούν στις τροποποιήσεις - επεκτάσεις που έκανες. Δες για παράδειγμα εδώ: https://github.com/courses-ionio/thesis/tree/master/current/2014221Κάνε ίσως και κάποια σχόλια για τα προβλήματα στον κώδικα που θα διορθώσεις σε επόμενο στάδιο. Μόλις γίνουν αυτά θα δεχτώ το pull request
 Τα υπόλοιπα τα λέμε στο αποθετήριό σου...

epidrome commented:
@merkourisa @c15pati θα πρέπει να βάλετε και μια αξιολόγηση για την τρέχουσα μορφή δεκτό-αλλαγές όπως η @bgarnb

epidrome commented:
@dafakias 
με την ευκαιρία της συζήτησης αυτής για το 3ο Παραδοτέο να προσθέσω ότι και το 2ο παραδοτέο, παρότι έγινε δεκτό, με δική μας ευθύνη φυσικά, δεν είναι πλήρες, γιατί θα πρέπει να περιέχει μια αναλυτική λίστα με προδιαγραφές όπως το ενδεικτικό που στείλαμε παραπάνω από συνάδελφο ή μερικά ακόμη από άλλες πτυχιακές σε εξέλιξη:
https://github.com/courses-ionio/thesis/tree/master/current/2011003
https://github.com/courses-ionio/thesis/tree/master/current/2012052
οπότε προτείνω με αυτήν την ευκαιρία να ενημερώσεις και το 2ο παραδοτέο γιατί χωρίς τις προδιαγραφές είναι δύσκολο να κάνουμε σχόλια και να βοηθήσουμε.

merkourisa reviewed (changes requested):
Εμπλούστισε την αναφορά σου με ενδεικτικές εικόνες. Διόρθωσε τα link στο repository του παιχνιδιού. Όσον αφορά το παιχνίδι ισχύει ότι σου είχα πει καλό θα ήταν αρχικά να φτιάξεις μια πίστα που παίζεται ευχάριστα και μετά να προσθέσεις το εκπαιδευτικό στοιχείο

dafakias commented:
Καλησπέρα, έχω δει όλα τα σχόλια σας οπότε θα κοιτάξω τις αναφορές των παιδιών που μου στείλατε και θα βελτιώσω την δική μου αναφορά με περισσότερες λεπτομέριες. Θα πρέπει να τα έχω έτοιμα μέχρι το βράδυ ή το αργότερο αύριο.

dafakias committed:
Update Readme.md

2ο-3ο ΠΑΡΑΔΟΤΕΟ 

Απλά μου εμφανίζει και τις προηγούμενες αλλαγές των συγκεκριμένων παραδοτέων, χωρίς να μπορώ να τις σβήσω.

dafakias commented:
Καλησπέρα, έχω ανεβάσει τις αλλαγές των παραδοτέων 2 και 3. Απλά όταν πάτησα preview, μου έβγαζε και τις προηγούμενες αλλαγές των συγκεκριμένων παραδοτέων χωρίς να μπορώ να τις σβήσω.

epidrome commented:
@dafakias ευχαριστούμε για την ενημέρωση!
[ισχύουν 100% τα σχόλια που έγραψα πριν](https://github.com/courses-ionio/thesis/pull/31#issuecomment-362998549)
η συμβουλή μου είναι να γίνει εκτενής συζήτηση με την συντονίστρια @bgarnb  της πτυχιακής στο ιδιωτικό σου αποθετήριο ή/και διαζώσης-σκαιπ έτσι ώστε να βγουν λεπτομερείς προδιαγραφές.
